liquidator/index.js

```diff
@@ -59,34 +65,32 @@ async function run(logger, address, pollingDelay, priceFeedConfig, liquidatorCon
     };
 
     // Setup price feed.
-    const getTime = () => Math.round(new Date().getTime() / 1000);
-
     const priceFeed = await createReferencePriceFeedForEmp(
       logger,
       web3,
       new Networker(logger),
       getTime,
-      address,
+      empAddress,
       priceFeedConfig
     );
 
-    if (!priceFeed) {
+    if (!priceFeedConfig) {
       throw new Error("Price feed config is invalid");
     }
 
     // Client and liquidator bot
     const empClient = new ExpiringMultiPartyClient(logger, ExpiringMultiParty.abi, web3, emp.address);
     const gasEstimator = new GasEstimator(logger);
-    const liquidator = new Liquidator(
+    const liquidator = new Liquidator({
       logger,
-      empClient,
-      voting,
+      expiringMultiPartyClient: empClient,
```

Comment:
this syntax I like by specifying the property passed in. should we not do this for all props, such as the `gasEstimator` and `priceFeed`?

Reply:
They're in a separate module, perhaps in another PR? :)